python/lsst/afw/table/baseColumnView.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+import numpy as np
+
+from ._baseColumnView import *
+
+def _get(self, key):
```

Comment:
This is surprising code: defining a method of a class and then assigning it later. Why is it necessary to do it this way? If this is not normal pybind11 technique then a comment here would be much appreciated, since it's not obvious until much later in the file what's going on.